account_loan/model/account_loan.py

```diff
@@ -86,7 +86,6 @@ def _default_company(self):
         default=0.0,
         digits=(8, 6),
         help='Currently applied rate',
-        track_visibility='always',
```

Comment:
why do you remove? This has been put for a reason, and is that once the loan has been confirmed one can change the rate, and we better track the changes.

Reply:
Thank You @jbeficent for your notes, sorry about that but I have add full logging messages as you can see and I didn't think that rate field was changeable after posting, therefore this change would be reverted now .